BUILDING.md

```diff
@@ -122,14 +122,13 @@ Please refer to [RUNC.md](/docs/RUNC.md) for the currently supported version of
 You can build static binaries by providing a few variables to `make`:
 
 ```sh
-make EXTRA_FLAGS="-buildmode pie" \
-	EXTRA_LDFLAGS='-linkmode external -extldflags "-fno-PIC -static"' \
-	BUILDTAGS="netgo osusergo static_build"
+make STATIC=1
 ```
```

Comment:
Maybe better to clarify that this is valid only for 1.7 and later.

Reply:
Does the patch apply "cleanly" on the release/1.6 branch? (if so, that would make things a bit easier and I could already adjust my build-scripts)

Reply:
That'd be nice yes; IIUC (looking at the change), both the "old" and "new" way would be supported, so it should not be a breaking change.

Reply:
The backport PR is #7045.